CMakeLists.txt

```diff
@@ -249,7 +249,9 @@ if (DEBUG)
 else()
   add_definitions(-DNDEBUG)
   if (NOT MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3")
+    if (NOT CMAKE_CROSSCOMPILING)
+      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3")
```

Comment:
Should we add the `set(CMAKE_C_FLAGS` line here too?  (It can't hurt)

Reply:
Yes I can do that, it is not a problem.